src/extension.ts

```diff
@@ -133,6 +135,7 @@ export async function activate(context: vscode.ExtensionContext): Promise<Terraf
       await updateLanguageServer(manifest.version, lsPath);
       await clientHandler.startClient();
       vscode.commands.executeCommand('setContext', 'terraform.showModuleView', true);
+      vscode.commands.executeCommand('setContext', 'terraform.showProviderView', true);
```

Comment:
I didn't think ahead when I made the module view and name the context name a less specific value. Do you think we should change this to `showTreeView` and make both providers key off that? The original purpose of using the context value here is to ensure that we only activate them when the LS is enabled and ready, so there really isn't a need for two values.

Reply:
I like this idea, and I think there wouldn't be a case where we want to show just one of the views. So I'll rename the context value and remove the new one.